src/main/java/uk/gov/hmcts/reform/prl/services/citizen/CaseService.java

```diff
@@ -1132,4 +1141,43 @@ private List<CitizenDocuments> getStatementOfServiceDocuments(List<Element<StmtO
                          .build()
             ).toList();
     }
+
+    private void addOrderAdditionalDocumentsToOtherDocuments(CaseData caseData,
+                                                             List<CitizenDocuments> citizenOrders,
+                                                             List<CitizenDocuments> otherDocuments) {
+        if (null != caseData.getManageOrders()
+            && CollectionUtils.isNotEmpty(caseData.getManageOrders().getAdditionalOrderDocuments())) {
+            otherDocuments.addAll(
+                citizenOrders.stream()
+                    .map(order -> getAdditionalDocuments(
+                        order,
+                        caseData.getManageOrders().getAdditionalOrderDocuments()
+                    ))
+                    .flatMap(Collection::stream)
+                    .toList());
+        }
+    }
+
+    private List<CitizenDocuments> getAdditionalDocuments(CitizenDocuments order,
+                                                          List<Element<AdditionalOrderDocument>> additionalOrderDocuments) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DD_MMM_YYYY_HH_MM_SS);
+
+        return additionalOrderDocuments.stream()
+            .map(Element::getValue)
+            .filter(addDoc -> getStringsSplitByDelimiter(addDoc.getServedOrders(), COMMA)
+                .contains(getOrderLabelForDynamicList(order)))
+            .map(addDoc -> addDoc.getAdditionalDocuments().stream()
+                .map(Element::getValue)
+                .map(document -> CitizenDocuments.builder()
+                    .document(document)
+                    .uploadedDate(LocalDateTime.parse(addDoc.getUploadedDateTime(), formatter))
```

Comment:
check with service team, if we need to show the other documents should have order made date instead of addtional documents uploaded date